perses/app/fah_generator.py

```diff
@@ -240,6 +251,8 @@ def run_neq_fah_setup(ligand_file,
                       rmsd_restraint=True,
```

Comment:
The docstring says this is default False, but its not set as False here.

Reply:
Fixed, thanks!